distribution/docs/src/main/resources/content/_managing/_running/troubleshooting.adoc

```diff
@@ -79,4 +80,18 @@ This can be caused when another ${branding} is not properly shut down.
 * Verify running java.exe are not ${branding} (e.g., kill/close if necessary).
 * Utilize automated start/stop scripts to run ${branding} as a service.
 
+2+^|*Troubleshooting Certificates*
+
+.6+.^|Connection errors, for example:
+
+* `java.net.SocketException: Broken pipe`
+* `java.net.SocketException: Connection reset`
+
+|Ensure all the CAs and the whole trust chain (all intermediates are included in the walk-up to the CA) have been added to the <<{managing-prefix}creating_a_new_keystore_truststore_with_an_existing_certificate_and_private_key,trust store>>.
+|Ensure none of the certs or CAs have expired.
```

Comment:
❓For the sake of consistency, should we always spell out certificate instead of using cert?

Reply:
yes